ruma-events/README.md

```diff
@@ -13,3 +13,193 @@ ruma-events requires Rust 1.40.0 or later.
 ## Documentation
 
 ruma-events has [comprehensive documentation](https://docs.rs/ruma-events) available on docs.rs.
+
+# How-To
+
+## Deserializing
+
+Ruma offers the `EventJson` wrapper to enable passing around JSON text that is semi validated. This
+is useful when [extending ruma](TODO_link). All event structs and enums implement `Serialize/Deserialize` so
+EventJson is not strictly needed anymore (pre 0.22 this was not the case).
```

Comment:
This may be a bit misleading, for homeserver that want to support federation, it is strictly required to have a mechanism like that.